### DISCUSSION TITLE
"School email must be verified on your Github Account" although email address is verified

### DISCUSSION THREAD

**Discussion by 99991** [community/community#137331] [GitHub Education]
I encountered the following bug when trying to sign up for GitHub Education. After filling out the application form and clicking "Continue", I got the error message "School email must be verified on your GithubAccount".

![error](https://github.com/user-attachments/assets/5aa144f1-75bc-45dc-bf57-70f67babafcc)

Just below, it said that I already had verified my two email addresses.

![verified](https://github.com/user-attachments/assets/7a9fe853-84d9-4886-9f0d-e5dc161d38e3)

The reason for that bug is that my first email contained a `+`, which is not parsed correctly by GitHub Education, resulting in this confusing error message. After deleting the verified email address with the `+` from "Profile Picture (Top Right) -> Settings -> Emails", I was able to proceed with my other verified email address.

I have already contacted support about this and have been reassured that this issue will be forwarded to the team to see how to fix it, but I decided to also post it here in case someone else has the same issue.

**Comment by k0range** [comment]
I had the same problem and this way solved it.
Thank you so much for this!

**Comment by sueWashed** [comment]
they really need to fix this issue since i cannot sign up!

**Reply by roxylius** [reply]
still isn't fixed. --_--

**Comment by Linux0Hat** [comment]
Thanks you so much it's worked for me !

**Comment by bhartm3** [comment]
I have the same issue, I dont have a `+` in the email, but a `.` . The email is verified but I can't submit an application.
